recipe/meta.yaml

```diff
@@ -32,9 +38,7 @@ test:
     - gif2rgb.exe -h  # [win]
```

Comment:
Can you remove these lines specific for win and make `gif2rgb -h` be used in unix and win?

Reply:
done